src/proxy-ui-api/frontend/src/views/AppLogin.vue

```diff
@@ -265,16 +266,21 @@ export default (
         });
     },
 
-    async fetchCurrentSecurityServer() {
+    fetchCurrentSecurityServer() {
       this.$store.dispatch('fetchCurrentSecurityServer').catch((error) => {
         this.$store.dispatch('showError', error);
       });
     },
-    async fetchSecurityServerVersion() {
+    fetchSecurityServerVersion() {
       this.$store
```

Comment:
```suggestion
    return  this.$store
```

Reply:
Refactored to an async function with blocking vuex dispatch